tests/test_models.py

```diff
@@ -0,0 +1,27 @@
+from djangocms_versioning_filer.models import copy_file
+
+from .base import BaseFilerVersioningTestCase
+
+
+class FilerCopyFileMethodTests(BaseFilerVersioningTestCase):
+
+    def test_copy_text_file(self):
+        file = self.create_file_obj(
+            original_filename='test.txt',
+            content='some text',
+            is_public=True,
+            owner=self.superuser,
+            folder=self.folder,
+        )
+        copy = copy_file(file)
+        self.assertTrue(file.file.storage.exists(file.file.path))
```

Comment:
I will do `file.refresh_from_db()` after copy_file and also check if paths are not the same. The rest is good.